src/lib/MVC/Symfony/Controller/Content/PreviewController.php

```diff
@@ -112,8 +113,9 @@ public function previewContentAction(
         }
 
         try {
+            $viewType = $request->query->get('viewType') ?? ViewManagerInterface::VIEW_TYPE_FULL;
```

Comment:
A code style nitpick:
```suggestion
            $viewType = $request->query->get('viewType', ViewManagerInterface::VIEW_TYPE_FULL);
```
I consider it a code style nitpick because the end effect is exactly the same.

Reply:
> A code style nitpick:
> 
> I consider it a code style nitpick because the end effect is exactly the same.

a5154fe922